test/integration/guest_can_change_quanity_in_cart_test.rb

```diff
@@ -0,0 +1,43 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.